src/IndexResolver/ProductIndexesResolver.php

```diff
@@ -0,0 +1,33 @@
+<?php
```

Comment:
Maybe this should be renamed to `ProductIndexScopesProvider`?

Reply:
Yes, it makes sense